torchtext/experimental/vectors.py

```diff
@@ -0,0 +1,21 @@
+import torch
+
+
+class Vectors(object):
```

Comment:
file factory function (write this in Python for now) - check for support file formats.
Example uses white space separated entries
```
def vectors_from_tsv_file(path):
    tuples = map(lambda x: (x.split()[0], torch.tensor(x.split()[1:], dtype=torch.float)) open(path).read())
    words = map(lambda x: x[0], tuples)
    vectors = map(lambda x: [1], tuples)
    return Vectors(words, vectors)
```

Potential future factory functions
vectors_from_url (folds download + extract_archive if necessary)

Later on we could add a dictionary of pretrained vectors with associated URL, which is currently [folded into vocab.py](https://github.com/pytorch/text/blob/master/torchtext/vocab.py#L469).

To get vectors from the internet people could then do

vectors = vectors_from_url(torchtext.experimental.vectors.URLs['Glove'])

Reply:
Created the factory method for creating vectors to files. Lmk what you think

Reply:
Change ```class Vectors(object)``` to inherit from ```torch.nn.Module```.